lib/TPP/Dialect/Tpp/TppUtils.cpp

```diff
@@ -337,10 +337,12 @@ bool canMapToTppRelu(linalg::GenericOp linalgOp) {
 }
 
 // Return true if the linalg.generic can be mapped to a tpp.relu.
+// TODO: check the output bbarg to not be used.
 bool isTppRelu(linalg::GenericOp linalgOp) {
   if (linalgOp.getNumLoops() != linalgOp.getNumParallelLoops())
     return false;
-  if ((linalgOp.getNumDpsInputs() != 0) || (linalgOp.getNumDpsInits() != 1))
+  if (/*(linalgOp.getNumDpsInputs() != 1) || */ (linalgOp.getNumDpsInits() !=
```

Comment:
Please avoid commented out code. Remove the code and add a comment if needed.

Reply:
done.